sdcm/utils/es_index.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import json
+import sys
+import os
+
+from typing import Any
+from pathlib2 import Path
+
+from sdcm.es import ES
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def create_index(index_name: str, mappings: dict, **kwargs) -> bool:
+    es_client = ES()
+    doc_type = kwargs.get("doc_type")
+    status = False
+    try:
+        es_client.indices.create(index=index_name)
+        es_client.indices.put_mapping(index=index_name, doc_type=doc_type, body=mappings,  # pylint: disable=unexpected-keyword-arg
+                                      include_type_name=True)
+        status = True
+    except Exception as exc:  # pylint: disable=broad-except
+        LOGGER.error("Create index failed with error %s", exc)
+
+    return status
+
+
+def get_mapping(mapping_filepath: str) -> Any:
+    filepath = Path(mapping_filepath)
+    if not filepath.exists():
+        LOGGER.error("Provided mapping file %s does not exists", mapping_filepath)
+        return None
```

Comment:
why return None ? and not raise exception ?

Reply:
fixed